src/AddressCard.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { bonds, InlineAccount, InlineBalance, AccountIcon, AddressLabel, AccountLabel, EtherBalance, CoinList } from 'parity-reactive-ui';
```

Comment:
Always order imports alphabetically and values in those imports alphabetically as well. Typically oder is -

1. External packages
2. <blank line>
3. Our packages
4. <blank line>
5. Local dependencies